app/assets/stylesheets/themes/scinote.scss

```diff
@@ -514,6 +514,10 @@ a[data-toggle="tooltip"] {
 
   > .sci-btn-group {
     margin-right: 30px;
+
+    form.button_to {
```

Comment:
Will it affect anything else?

Reply:
I don't think so, since it's scoped specifically to .tab-pane-protocols